README.md

```diff
@@ -9,10 +9,16 @@ This repo is owned and maintained by the Zendesk Apps team. You can reach us on
 ## Getting Started
 When you want to help **develop** this tool, you will need to clone this repo.
 
+The Bundler Gem Version 1.17.3 is required for bundling dependencies. If you are working with a version that is higher than `1.17.3`, you will need to downgrade it to work on ZAS.
```

Comment:
Let's give a reason here, e.g. "Since ZAS is used in ZAT, which supports Ruby 2.1, the Bundler Gem Version...."

Reply:
Oh that sounds GOOD!

Reply:
This will now be
```
Since ZAS is used in ZAT, which supports Ruby 2.1, the Bundler Gem Version 1.17.3 is required for bundling dependencies. If you are working with a version that is higher than `1.17.3`, you will need to downgrade it.
```